src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -480,9 +480,13 @@ namespace wallet_rpc
       std::list<transfer_destination> destinations; // Array of destinations to receive LOKI.
       uint32_t account_index;                       // (Optional) Transfer from this account index. (Defaults to 0)
       std::set<uint32_t> subaddr_indices;           // (Optional) Transfer from this set of subaddresses. (Defaults to 0)
-      uint32_t priority;                            // Set a priority for the transaction. Accepted Values are: default (1), or 0-3 for: unimportant, normal, elevated, priority.
-      bool blink;                                   // Specifies that the tx should be blinked (`priority` will be ignored).
+      uint32_t priority;                            // Set a priority for the transaction. Accepted Values are: 0-5 for: default, unimportant, normal, elevated, priority, blink.
+#if LOKI_DEPRECATE_AT_VERSION(7, 1, 0)
+      bool blink;                                   // (Deprecated) Specifies that the tx should be blinked (`priority` will be ignored).
       uint64_t ring_size;                           // (Deprecated) Set to 10. Sets ringsize to n (mixin + 1). Loki ring_size is statically set to 10.
```

Comment:
No need to future-deprecate this rather than just straight up removing the field.